text/0000-otlp-http-json.md

```diff
@@ -0,0 +1,146 @@
+# OTLP: HTTP+JSON Transport Extension for OTLP
```

Comment:
Nit: this is not a transport extension but an additional encoding type for existing OTLP/HTTP transport.
Perhaps reword slightly:
```suggestion
# OTLP: JSON Encoding for OTLP/HTTP
```